nautobot/extras/tests/test_customfields.py

```diff
@@ -976,16 +977,40 @@ def test_custom_choice_deleted_with_field(self):
         self.assertEqual(CustomFieldChoice.objects.count(), 0)
 
 
-class CustomFieldBackgroundTasks(TestCase):
-    def setUp(self):
-        # Clear the queue for each test
-        django_rq.get_queue("custom_fields").empty()
+class CustomFieldBackgroundTasks(TransactionTestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Start a celery worker"""
+        super().setUpClass()
+        app.loader.import_module("celery.contrib.testing.tasks")
```

Comment:
How's this different than just doing `import celery.contrib.testing.tasks`? Mind adding an explanatory comment?

Reply:
It is something to do with the way that the loader class operates. It is registering several tasks like `ping` which are required to actually bring the worker up in this mode.